compiler/src/iree/compiler/Codegen/Common/CPU/CPUMaterializeEncodingPass.cpp

```diff
@@ -49,10 +49,7 @@ enumerateMatmulTilesGeneric(EncodingUser user, ExecutableTargetAttr target) {
   }
 
   return {
-      TileMxNxK{8, 8, 4}, // Some vaguely reasonable static tile shape.
-      TileMxNxK{4, 8, 4}, // Truncation of the above.
-      TileMxNxK{2, 8, 4}, // Truncation of the above.
-      TileMxNxK{1, 8, 4}, // Truncation of the above.
```

Comment:
Should we keep these configurations? If not, can you update the function comment above?